asyncio_mqtt/client.py

```diff
@@ -218,6 +227,18 @@ async def force_disconnect(self) -> None:
         if not self._disconnected.done():
             self._disconnected.set_result(None)
 
+    def _outgoing_call(self, func: Callable) -> Callable[..., Awaitable]:
+        async def decorated(*args: Any, timeout: int = 10, **kwargs: Any) -> None:
```

Comment:
Why is `timeout` mentioned explicitly here? Let's just omit it (let it go into the `kwargs`).

Reply:
Also, let's add [`functools.wraps` to preserve the attributes of the original function](https://docs.python.org/3/library/functools.html#functools.wraps) (`__name__`, `__doc__`, etc.).

Reply:
I included it to more explicitly mirror the method declarations that it's wrapping – example:

https://github.com/sbtinstruments/asyncio-mqtt/blob/2a352ba87c35d4c4ee22fa4420823828e1fff025/asyncio_mqtt/client.py#L243

That said, happy to simplify if that's what you'd prefer.

Reply:
Let's simplify it away then. 👍 Shorter upgrade path towards `ParamSpec` in 3.10. Similar "type scope" arguments as for the return type (see the other comment).